### PR TITLE
Reverts Backtrace: format to what it used to be (IDFGH-7580)

### PR DESCRIPTION
**Pull request by MitchBradley**
Was:  Backtrace: N:M N:M N:M ...
Now:  Backtrace:N:MN:M N:M ...

The problem with the new format is that it is hard to parse and
breaks the parser that is used by PlatformIO.  The old format
is much more reasonable.  I do not see how the pattern in IDFDUT.py
can work with the new format, due to the missing space after the :

Possible fix for #8722

### CODE REVIEW

**Comment by Alvin1Zhang** [comment]
Thanks for your contribution.

**Comment by SoucheSouche** [comment]
Hello @MitchBradley,
The backtrace parsing issue you mentioned is going to be fixed using the proposed changes.
I will keep you informed of the state of the PR.

**Comment by SoucheSouche** [comment]
Hello @MitchBradley,
The changes were merged internally and will be available on GitHub in the next commit on the master branch.
This could take from  a couple of days to a couple of weeks.
Thanks again for your contribution.
